v22.1/build-a-typescript-app-with-cockroachdb.md

```diff
@@ -84,6 +86,11 @@ This tutorial shows you how run a simple application built with [TypeORM](https:
 
```

Comment:
the routing id param looks like it's just called "Options" in the console. might be good to clarify that... or change the label in the console.
<img width="573" alt="Screen Shot 2022-04-15 at 12 02 33 PM" src="https://user-images.githubusercontent.com/75328174/163593486-180fa8f5-cede-4e00-a474-b0fcdd3c5a6f.png">
.